src/txorphanage.cpp

```diff
@@ -102,31 +102,31 @@ void TxOrphanage::EraseForPeer(NodeId peer)
     if (nErased > 0) LogPrint(BCLog::MEMPOOL, "Erased %d orphan tx from peer=%d\n", nErased, peer);
 }
 
-unsigned int TxOrphanage::LimitOrphans(unsigned int max_orphans)
+void TxOrphanage::LimitOrphans(unsigned int max_orphans)
 {
     AssertLockHeld(g_cs_orphans);
 
-    unsigned int nEvicted = 0;
+    unsigned int nExpired = 0;
     static int64_t nNextSweep;
     int64_t nNow = GetTime();
     if (nNextSweep <= nNow) {
         // Sweep out expired orphan pool entries:
-        int nErased = 0;
         int64_t nMinExpTime = nNow + ORPHAN_TX_EXPIRE_TIME - ORPHAN_TX_EXPIRE_INTERVAL;
         std::map<uint256, OrphanTx>::iterator iter = m_orphans.begin();
         while (iter != m_orphans.end())
         {
             std::map<uint256, OrphanTx>::iterator maybeErase = iter++;
             if (maybeErase->second.nTimeExpire <= nNow) {
-                nErased += EraseTx(maybeErase->second.tx->GetHash());
+                nExpired += EraseTx(maybeErase->second.tx->GetHash());
             } else {
                 nMinExpTime = std::min(maybeErase->second.nTimeExpire, nMinExpTime);
             }
         }
         // Sweep again 5 minutes after the next entry that expires in order to batch the linear scan.
         nNextSweep = nMinExpTime + ORPHAN_TX_EXPIRE_INTERVAL;
-        if (nErased > 0) LogPrint(BCLog::MEMPOOL, "Erased %d orphan tx due to expiration\n", nErased);
+        if (nExpired > 0) LogPrint(BCLog::MEMPOOL, "Erased %d orphan tx due to expiration\n", nExpired);
```

Comment:
The current name isn't optimal. There are two options here:

* You don't change the name of `nErased` to `nExpired`, because it is not needed.
* You change the name to `num_expired`, because the style guide recommends that for new symbols.

Reply:
I prefer the 1st option. Since other functions in `txorphanage` also have camelCase symbols, its better to change them all in a `style:` commiy.